src/test/java/org/kcctl/IntegrationTest.java

```diff
@@ -93,7 +112,28 @@ public void cleanup() {
     @AfterAll
     public static void resetKafkaConnectImage() {
         // In case a test overrode the Kafka Connect image
-        kafkaConnect = kafkaConnectBase;
+        kafkaConnect = kafkaConnectLatestStable;
+    }
+
+    /**
+     * Run a command with the given arguments and verify that it has exited successfully
+     * @param context the {@link KcctlCommandContext} for the command
+     * @param args the arguments with which the command should be invoked
+     */
+    protected static void runAndCheckExitCode(KcctlCommandContext<?> context, String... args) {
```

Comment:
This was helpful when debugging the test failures which were only cropping up when multiple integration test suites were run (which made it difficult to run things through a debugger in my IDE).

Similar to the varargs variant of `registerTestConnectors`, if this looks good, can adapt the rest of the integration test code base to use it wherever applicable.

Reply:
Tweaked this method to be a public instance method for `KcctlCommandContext` instead of a protected static method for `IntegrationTest`, but the general idea is the same.

Reply:
Applied this change across the board in https://github.com/kcctl/kcctl/pull/354